changelogs/client_server/newsfragments/1306.clarification

```diff
@@ -0,0 +1 @@
+Replace various instances of the word "plaintext" with "cleartext".
```

Comment:
I'd go with a more generic

```suggestion
Various clarifications throughout the specification.
```

Reply:
I like that better than the generic "Fix various typos throughout the specification." we've been using elsewhere.